Modules/Core/Http/Controllers/BasePublicController.php

```diff
@@ -13,10 +13,25 @@
      */
     protected $auth;
     public $locale;
+    public $alternateUrls = [];
 
     public function __construct()
     {
         $this->locale = App::getLocale();
         $this->auth = app(Authentication::class);
+        view()->share('alternate', $this->alternateUrls);
     }
+
+    /**
+     * Add alternate URLs to main array and inject it to the page
+     *
+     * @param array $alternateUrls
+     * @return void
+     */
+    protected function addAlternateUrls(array $alternateUrls)
+    {
+        $this->alternateUrls = array_merge($this->alternateUrls, $alternateUrls);
+        view()->share('alternate', $this->alternateUrls);
+    }
+}
```

Comment:
I just noticed this. There's an extra closing bracket here.

Reply:
Sent a pull request to remove it